config/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "blogger",
```

Comment:
can we just come up with a name for the database? I suggest something like "wingman_db"

Reply:
You're absolutely right, I should've changed that already. I'm fine with your suggestions of "wingman_db". We can confirm it tonight and then push the change.